### PR TITLE
#8764: More docs changes for WH readiness, Part 5

### PR DESCRIPTION
**Pull request by tt-rkim**
### Ticket
https://github.com/tenstorrent/tt-metal/issues/8764

### Problem description
Getting documentation ready for Wormhole readiness, with an emphasis of what works on N150/N300.

### What's changed

Change documentation based on customer team feedback.

### Checklist
- [x] Post commit CI passes
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

**Comment by tt-rkim** [comment]
@dvartaniansTT did we verify the new wheel flow?

File: INSTALLING.md [hunk 1]
```diff
@@ -90,39 +94,31 @@ cd tt-metal
...
 ```
 
-Note about Python environments: You do not have to use `create_venv.sh`. If you
-are less familiar with Python and its various environment tools, just use
-`create_venv.sh` as shown above and the pre-built environment.
+where `ARCH_NAME` is either `grayskull` or `wormhole_b0`, depending on your Tenstorrent device.
+
+> [!NOTE]
+>
+> Note about Python environments: You do not have to use `create_venv.sh`. If you are less familiar with Python and its various environment tools, just use `create_venv.sh` as shown above and the pre-built environment.
```
Comment (dvartaniansTT) [inline INSTALLING.md:121]: I still need to reproduce the errors when building outside of the python virtual env. Will have it sometime tomorrow.

File: INSTALLING.md [hunk 2]
```diff
@@ -139,13 +135,14 @@ pip install <wheel_file.whl>
 ```
 
 If you are going to try our pre-built models in `models/`, then you must also
-further install their requirements:
+further install their required dependencies and environment variables:
 
 ```sh
+export PYTHONPATH=$(pwd)
```
Comment (dvartaniansTT) [inline INSTALLING.md:141]: people might try to build from source first and they may have already exported the env variables relevant to source build which will break the whl build. 

good idea to let user know to undo the env variables set for source build if they are going to try whl build next.

**Comment by tt-rkim** [comment]
The understanding is that we're putting the installation instructions on hold until @dvartaniansTT has time to try it out.

I may merge the non-installation instruction changes. 

@dvartaniansTT @mbahnasTT

**Pull request merged**